test/integration/e2e/end_to_end.go

```diff
@@ -125,7 +128,11 @@ func createChannelAndCC(t *testing.T, sdk *fabsdk.FabricSDK) {
 	}
 
 	// Create chaincode package for example cc
-	createCC(t, orgResMgmt)
+	if metadata.Ccmode == "Lscc" {
```

Comment:
Also - I don't think I see the "Lscc" codepath currently exercised in this PR.
(but it should be exercised with prev tests as previously mentioned in prior comment).